### PR TITLE
Set up Krypto module for use in KPlutus

### PR DESCRIPTION
**Pull request by gtrepta**
fixes: #87

### CODE REVIEW

**Review by gtrepta — COMMENTED** [review]
The test program `tests/simple/delay.uplc`:
```
(program 1.0.0 (delay x))
```
Is using the identifier `x`. However, in `krypto.md` there's the syntax declaration
```
syntax G2Point ::= "(" Int "x" Int "," Int "x" Int ")"
```
Which was causing a parsing error on the `x` token.

This commit which uses `--syntax-module UPLC-SYNTAX` fixes that issue. I'm making a note of it here because it might not be the preferred solution. We could also change the test programs to not use `x`, though it's a pretty common identifier and could lead to some perplexing errors down the road for people who try to use it.

File: Makefile [hunk 1]
```diff
@@ -135,7 +135,7 @@ $(KPLUTUS_INCLUDE)/kframework/%.md: %.md
 
 llvm_dir           := llvm
 llvm_main_module   := UPLC
-llvm_syntax_module := $(llvm_main_module)
+llvm_syntax_module := UPLC-SYNTAX
```
Comment (gtrepta) [inline Makefile:138]: I meant to reference this change here in my comment above ^^^

**Comment by ChristianoBraga** [comment]
It looks great to me!

**Review by ehildenb — APPROVED** [review]

**Review by ChristianoBraga — APPROVED** [review]
It looks great to me.

**Comment by ehildenb** [comment]
Looks good to me too! I guess we should change the syntax of `G2Point` to be something like `G2Point(Int, Int, Int, Int)` instead, because that is confusing, but the syntax module change would probably have been needed later anyway!

**Pull request merged**